app/src/lib/stores/app-store.ts

```diff
@@ -1624,36 +1613,22 @@ export class AppStore extends TypedBaseStore<IAppState> {
   }
 
   private startPullRequestUpdater(repository: Repository) {
-    if (this.currentPullRequestUpdater) {
-      this.stopPullRequestUpdater()
```

Comment:
The removal of this `stop` operation from the `startPullRequestUpdater` method makes me a teeny bit nervous. I can' point towards anything that will break by removing this but given that `pullRequestCoordinator.stopPullRequestUpdater` is both cheap and idempotent could we maintain the current behavior just to be safe?

Reply:
ah I see. even though we do this in `PullRequestCoordinator.startPullRequestUpdater`, it might not get called! fixed in https://github.com/desktop/desktop/pull/8825/commits/8fb41ce0e2e7201b1065487d3cae512c5287fe0c